index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Dunk card display</title>
+		<link rel="stylesheet" type="text/css" href="quize.css">
+	</head>
+	<body>
+
+		<article1 class="box">
+			<section class="section1">
+				<h2>Yellow Dunk Cards</h2>
+			</section>
+
```

Comment:
Requirement 3 specifies "Each product should contain a `<header>` element that, itself, contains an `<h1>` element where the product's title will be written." So this should be in a header and an h1, not a section and h2.